docs/compatible_microcontrollers.md

```diff
@@ -62,6 +62,52 @@ You can also use any ARM chip with USB that [ChibiOS](https://www.chibios.org) s
  * [MK66FX1M0](https://www.nxp.com/products/processors-and-microcontrollers/arm-microcontrollers/general-purpose-mcus/k-series-cortex-m4/k6x-ethernet/kinetis-k66-180-mhz-dual-high-speed-full-speed-usbs-2mb-flash-microcontrollers-mcus-based-on-arm-cortex-m4-core:K66_180)
    * PJRC Teensy 3.6
 
+### Raspberry PI RP2040
+
+The following drivers are currently supported on RP2040 MCUs.
+
+| System           | Support                                       |
+| ---------------- | --------------------------------------------- |
+| ADC driver       | Support planed                                |
+| Audio            | Support planed                                |
+| I2C driver       | :heavy_check_mark:                            |
+| SPI driver       | :heavy_check_mark:                            |
+| WS2812 driver    | :heavy_check_mark: using `PIO` driver         |
+| External EEPROMs | :heavy_check_mark: using `I2C` + `SPI` driver |
+| EEPROM emulation | Support planed                                |
+| serial driver    | :heavy_check_mark: using `SIO` + `PIO` driver |
+| UART driver      | Support planed                                |
+
+#### GPIO pin nomenclature
+
+To address individual pins QMK uses the `GPx` abbreviation, the `x` stands for the GPIO number of the pin. This can usually be found on the official pinout diagramm of your board. Please note that these GPIO numbers not necessarily match the number you see printed on the development board. For instance the [Raspberry Pi Pico](https://datasheets.raspberrypi.com/pico/Pico-R3-A4-Pinout.pdf) uses numbers from 1 to 40 for their pins, but these are not the internal GPIO numbers. So if you want to use the pin 11 of the Pico for your keyboard, you would reefer to it as `GP8` in the config files.
```

Comment:
> This can usually be found on the official pinout diagramm of your board.

Do you mean that the RP2040 GPIO numbers are different from the Pico pinout numbers?
Maybe link to the RP2040 pinout https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf on page 13.

And together with the Pico pinout, a popular one would be the Adafruit KB2040 https://learn.adafruit.com/assets/106984

Reply:
> Do you mean that the RP2040 GPIO numbers are different from the Pico pinout numbers?

Yes, check the pin numbering on the linked image in the nomenclature. The first grey number is what the documentation is talking about.

>Maybe link to the RP2040 pinout https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf on page 13.

Is this necessary? 

The Pico seems to be the most standard configuration for prepackaged controllers, so a pinout link in documentation would be helpful. Someone using a different pinout (KB2040, or even just the bare chip) can just look that up for themselves

Reply:
Thanks! I didn't even notice the gray number. I alway look at the "colored" labels, and those should be the same between RP2040, Pico, KB2040 (and hopefully others too).

Reply:
Alright, that definitely showed the ambiguity in the current documentation. I think a directly embedded picture with the pinout of the pico and perhaps the kb2040 as most designs are done around a avr pro micro would be helpful at this point.

Reply:
Updated the documentation with pinouts of the pico and kb2040